core-web/libs/ui/src/lib/components/dot-empty-container/dot-empty-container.component.ts

```diff
@@ -37,6 +37,9 @@ export class DotEmptyContainerComponent {
     @Input()
     buttonLabel: string;
 
+    @Input()
```

Comment:
add a small doc related of what do this input like the other inputs.

Reply:
it is, because is happening in the `ngOnInit`